platform/mellanox/mlnx-ssd-fw-update.sh

```diff
@@ -21,9 +21,8 @@
 #=  Global variable                                                            #
 #=
 #=====
-VERSION="1.5"
+VERSION="1.6"
```

Comment:
why do we need to change the ssd upgrade? iMO it was supposed to be the same. can you please elaborate?

Reply:
This file is provided by low-level team. it needs to be updated to support SSD upgrade for nvme disk because the way to look for ssd's dev name is changed.